roles/cinder-data/tasks/main.yml

```diff
@@ -1,36 +1,18 @@
 ---
-- name: install cinder-data required packages
-  package: name={{ item }}
-  with_items: "{{ cinder.data_pkgs[ursula_os] }}"
-  register: result
-  until: result|succeeded
-  retries: 5
+- include: lvm_integration.yml
+  when: not ceph.enabled
```

Comment:
we cannot assume that when ceph is not enabled it will be lvm. Need feature flag when: lvm.enabled